docs/development/go/go-context.md

```diff
@@ -0,0 +1,468 @@
+
+---
+author:
+  name: Mihalis Tsoukalos
+  email: mihalistsoukalos@gmail.com
+description: 'Showcasing the use of the context Go package.'
+keywords: ["go", "golang", "context"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2019-05-31
+modified_by:
+  name: Linode
+title: 'Using the context Go package'
+contributor:
+  name: Mihalis Tsoukalos
+  link: https://www.mtsoukalos.eu/
+external_resources:
+  - '[Go](https://www.golang.com)'
+---
+
+## Before You Begin
+
+You will need to install a recent version of Go on your computer in order to follow the
+presented commands. Any Go version newer than 1.8 will do but it is considered a good
+practice to have the latest version of Go installed. You can check your Go version
+by executing `go version`.
+
+If you still need to install Go, you can follow our guide for Ubuntu installation [here](https://www.linode.com/docs/development/go/install-go-on-ubuntu/).
+
+{{< note >}}
+This guide is written for a non-root user. Depending on your configuration, some commands might require the help of `sudo` in order to get property executed. If you are not familiar with the `sudo` command, see the [Users and Groups](/docs/tools-reference/linux-users-and-groups/) guide.
+{{< /note >}}
+
+## About the context package
+
+The main purpose of the `context` package is to define the `Context` type and
+support *cancellation*. This happens because there are times where you want to
```

Comment:
This is partly correct, but not the full story: The `context` package supports both the handling of multiple concurrent operations and passing of (typically request-scoped) contextual data in key-value pairs.

Reply:
Used your description of `context`.